snapshot/lib/snapshot/reset_simulators.rb

```diff
@@ -80,6 +91,10 @@ def self.devices
       result.select { |parsed| parsed.length == 3 } # we don't care about those headers
     end
 
+    def self.runtimes
+      all_runtime_type = Helper.backticks('xcrun simctl list runtimes', print: FastlaneCore::Globals.verbose?).scan(/(.*)\s\(\d.*(com\.apple[^)\s]*)/)
```

Comment:
Don't know if it should be it's own method but otherwise I couldn't see how I should get the test to work.